packages/gatsby-theme/src/components/MobileMenu/MobileMenu.stories.mdx

```diff
@@ -53,23 +53,30 @@ export const BComponent = (args) => {
   )
 }
 
-<Meta title='components/MobileMenu' component={MobileMenu} />
+<Meta
+  title='components/MobileMenu'
+  component={MobileMenu}
+  subcomponents={{
+    MobileMenuItem: MobileMenu.Item,
+    MobileMenuNavbar: MobileMenu.Navbar,
+  }}
+/>
 
 # MobileMenu
 
-Use this MobileMenu
+Use this component to insert a MobileMenu.
 
-<Props of={MobileMenu} />
+<ArgsTable of={MobileMenu} />
 
 ### Default
 
-<Preview>
+<Canvas>
   <Story name='Default'>{BComponent.bind({})}</Story>
-</Preview>
+</Canvas>
 
 ### MegaMenu
 
-<Preview>
+<Canvas>
   <Story name='MegaMenu'>
     <MobileMenu showSidebar handleShowSidebar={() => {}}>
```

Comment:
I think the showSidebar prop is kind of breaking the `docs` on the MegaMenu and Default stories. I wasn't able to check the docs of neither of those stories while this property was enabled. 

I think it's being read while the docs page is loading and the MegaMenu is rendered on top of the docs page. I don't know if we could consider evaluating doing a refactor or some type of workaround to fix this issue. Perhaps we could consider removing the "MegaMenu" story? What do you think?